onmt/inputters/inputter.py

```diff
@@ -75,7 +75,7 @@ def save_fields_to_vocab(fields):
     vocab = []
     for k, f in fields.items():
         if f is not None and 'vocab' in f.__dict__:
-            f.vocab.stoi = dict(f.vocab.stoi)
+            f.vocab.stoi = f.vocab.stoi
```

Comment:
Does this line do anything? We needed to make it a dict before to save. Remove or check.

Reply:
@pltrdy any comment on this one ?
I know that we had crashing issue while the dict() was there.
to me it's unclear.